internal/config/config.go

```diff
@@ -751,6 +764,47 @@ func (c *Config) WithOpenStackProviderDefaults(openStackProvider string) *Config
 	return c
 }
 
+// AWSSEVSNP is the configuration for AWS SEV-SNP attestation.
+type AWSSEVSNP struct {
+	// description: |
+	//   Expected TPM measurements.
+	Measurements measurements.M `json:"measurements" yaml:"measurements" validate:"required,no_placeholders"`
+	// description: |
+	//   Expected launch measurement in SNP report.
+	LaunchMeasurement measurements.Measurement `json:"launchMeasurement" yaml:"launchMeasurement" validate:"required"`
```

Comment:
In the [attestation config RFC](https://github.com/edgelesssys/constellation/blob/main/rfc/attestation-config.md#aws), we specified the AWS SNP launch-measurement to be a list of acceptable values, this wont work if we use `measurement.Measurement` for the measurement in the config.
Instead, we should use a structure similar to the one used for ID key digest on Azure.

Reply:
If it is alright with you I would keep this as is for now. We are still not 100% sure how the final object might look like. Because we might write a port of [ibm/sev-snp-measure](https://github.com/IBM/sev-snp-measure). Or we might hit some roadblock while doing so. 

The upcoming attestation PR will also ignore the field in the beginning so we have something working quickly.